language-plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -117,7 +119,7 @@ genNatRoundtrip = do
         ssize = Constant () $ BuiltinSize () sizev
         typedIntSized = TypedBuiltinSized (SizeValue sizev) TypedBuiltinSizedInt
     TermOf _ iv <- Gen.filter ((>= 0) . _termOfValue) $ genTypedBuiltinDef typedIntSized
-    term <- lift $ do
+    term <- lift $ rename =<< do
```

Comment:
None of those silly `rename`s should be there, I'll remove them in a separate PR as it's unrelated here.